source/js/buyers-guide/components/creep-vote/creep-vote.jsx

```diff
@@ -52,9 +52,8 @@ export default class CreepVote extends React.Component {
         creepiness: creepinessId,
         confidence: confidence[0] > confidence[1] ? 0 : 1
       },
-      submitAttempted: false,
       subscribed,
-      nextView: subscribed ? "DidVote" : "SignUp",
+      showNewsletter: undefined,
```

Comment:
Normally we don't assign `undefined` to a variable as it's already `underfined` by default. Let's set this to `false`.

Reply:
Sorry about that, I meant to update it `false` 🙈